server/bundler/loader.js

```diff
@@ -113,9 +113,10 @@ function splitTemplate( path, section ) {
 		'		next();',
 		'	}, function onError( error ) {',
 		'		if ( ! LoadingError.isRetry() ) {',
+		'			console.warn( error );',
```

Comment:
I've added that debug as I feel that it could help us to spot other similar issues.